docs_src/cors/tutorial001.py

```diff
@@ -14,8 +14,8 @@
     CORSMiddleware,
     allow_origins=origins,
     allow_credentials=True,
-    allow_methods=["*"],
-    allow_headers=["*"],
+    allow_methods=["GET"],
+    allow_headers=["X-Example"],
```

Comment:
```suggestion
    allow_headers=["Authorization", "X-Example", "Content-Type"],
```
I think it would make sense to add `Authorization` header (as we specified `allow_credentials=True`) and probably `Content-Type` header (as it's very common).
What do you think?

Reply:
Thank you for taking the time to review this PR.

Yes, that makes sense; though since this example is a `simple request`, the `Content-Type` header is allowed by default, the docs mention that [here](https://github.com/fastapi/fastapi/blob/master/docs/en/docs/tutorial/cors.md?plain=1#L59C1-L59C75). I don't think we have to explicitly add it as it might seem a bit confusing on a first read regarding why we did that for `Content-Type` and not the other defaults.

What do you think? I'm happy to add it if you believe it would be helpful to have it there.

Reply:
It's true for simple requests. But if you send a request with credentials, it will not be a simple request

Reply:
@simone-trubian, good point :)

```
app.add_middleware(
    CORSMiddleware,
    allow_origins=origins,
    allow_credentials=False,
    allow_methods=["*"],
    allow_headers=["*"],
)
```

Reply:
Yes, that's a good point. I will update the files; thanks for the review guys.